avro-codegen/src/test/java/com/linkedin/avroutil1/codegen/SpecificRecordClassGeneratorTest.java

```diff
@@ -77,23 +76,38 @@ public void testSimpleFixedWithHugeDoc() throws Exception {
     Assert.assertNull(result.getParseError());
     AvroFixedSchema fixedSchema = (AvroFixedSchema) result.getTopLevelSchema();
     Assert.assertNotNull(fixedSchema);
-    for (JavaFileObject javaFileObject : generator.generateSpecificClassWithInternalTypes(fixedSchema,
-        SpecificRecordGenerationConfig.BROAD_COMPATIBILITY)) {
-      CompilerHelper.assertCompiles(javaFileObject);
-    }
+    JavaFileObject javaFileObject =
+        generator.generateSpecificClass(fixedSchema, SpecificRecordGenerationConfig.BROAD_COMPATIBILITY);
+    CompilerHelper.assertCompiles(javaFileObject);
+
+  }
+
+  @Test
+  public void testRecordWithArrayOfRecords() throws Exception {
```

Comment:
nit - method name doesnt match whats actually in the record (its an array of string)